README.md

```diff
@@ -44,7 +44,271 @@ To build, ensure docker is installed, and run:
 
 ### Deploy a GD2 gluster cluster
 
-### Deploy CSI driver
+### 1. RW0 Volume Claim (Gluster Virtual Block CSI driver)
+
+```
+[root@localhost]#kubectl create -f csi-deployment.yaml
```

Comment:
in which directory `csi-deployment.yaml` is present please update

Reply:
Done